packages/venia-ui/lib/components/FilterModal/CurrentFilters/currentFilter.js

```diff
@@ -9,15 +10,26 @@ import defaultClasses from './currentFilter.css';
 
 const CurrentFilter = props => {
     const { group, item, removeItem } = props;
+    const { formatMessage } = useIntl();
     const classes = mergeClasses(defaultClasses, props.classes);
 
     const handleClick = useCallback(() => {
         removeItem({ group, item });
     }, [group, item, removeItem]);
 
+    const ariaLabel = formatMessage(
+        {
+            id: 'filterModal.action.clearFilterItem.ariaLabel',
+            defaultMessage: 'Clear filter "{name}"'
```

Comment:
Also don't use variables in default messages. Just leave this as `Clear filter`. You can put the placeholder in the actual message in the json.